gnome/gingerblue/Portfile

```
@@ -23,11 +23,12 @@ master_sites        gnome:sources/${name}/${branch}/
 
 use_xz              yes
 
-checksums           rmd160  56a9c1b51324ad82395e683da17ff7164117b0f1 \
-                    sha256  dd63cd1442fd9b8c0c6b31a9c9dc68d9a992a72c0642b0d7942889d8eb280f27 \
-                    size    321940
+checksums           rmd160  b03f5c9fcf6c3ccc26f9aeb82749a63615216966 \
+                    sha256  074de41b083a4ea757d7cc48ff48cac2c2c318f600408262a185f2b222ba4931 \
+                    size    322640
 
 depends_build       port:gnome-common \
+                    port:desktop-file-utils
```

This line is a syntax error.
```suggestion
                    port:desktop-file-utils \
```
You will also need to rebase to remove the merge commit before we can accept this PR.

I see you've made your PR from master. You will have an easier time in the future if you make each PR from a new branch, not from master.